etl/steps/data/garden/wb/2024-03-11/income_groups.meta.yml

```diff
@@ -1,7 +1,40 @@
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Economic Growth
+    processing_level: major
+    description_from_producer: |-
+      For the current 2024 fiscal year, low-income economies are defined as those with a GNI per capita, calculated using [the World Bank Atlas method](https://datahelpdesk.worldbank.org/knowledgebase/articles/378832-what-is-the-world-bank-atlas-method), of $1,135 or less in 2022; lower middle-income economies are those with a GNI per capita between $1,136 and $4,465; upper middle-income economies are those with a GNI per capita between $4,466 and $13,845; high-income economies are those with a GNI per capita of $13,846 or more.
+
+      Income classifications are set each year on July 1 for all World Bank member economies, and all other economies with populations of more than 30,000. These official analytical classifications are fixed during the World Bank's fiscal year (ending on June 30), thus economies remain in the categories in which they are classified irrespective of any revisions to their per capita income data. The historical classifications shown are as published on July 1 of
+      each fiscal year.
+
+      Regions in this dataset include economies at all income levels. The term country, used interchangeably with economy, does not imply political independence but refers to any territory for which authorities report separate social or economic statistics. For more information about how the World Bank classifies countries, check [their documentation](https://datahelpdesk.worldbank.org/knowledgebase/articles/378834-how-does-the-world-bank-classify-countries).
+    description_key:
+      - The World Bank creates a yearly classification of countries by income, for all countries with population over 30,000.
+      - This classification stays the same throughout the fiscal year (from July 1 to June 30) even if the income data for a country changes.
+      - Low-income countries are those with a gross national income (GNI) per capita of $1,135 or less in 2022.
+      - Lower-middle-income countries are those with a GNI per capita between $1,136 and $4,465 in 2022.
+      - Upper-middle-income countries are those with a GNI per capita between $4,466 and $13,845 in 2022.
+      - High-income countries are those with a GNI per capita of $13,846 or more in 2022.
+
```

Comment:
Interesting.

I'd have put this in the snapshot description. I can see why you want to put it there. Is that something we want to do more often?

Reply:
I initially had this info only in the snapshot description. Then I copied it to key description. Then I removed it from snapshot description (not sure if this was a good decision). I think it would be ok to have this information in both places, but the key description is certainly more important, as it's surfaced in more prominent places.